### PR TITLE
fix date display, cheat 'win scenario' and position of 'end of new strin...

### PR DESCRIPTION
**Pull request by dkessel**
fix date display, cheat 'win scenario' and position of 'end of new strings' marker in german localisation

### CODE REVIEW

File: data/language/german.txt [hunk 1]
```diff
@@ -2769,7 +2769,7 @@ STR_2762    :Bezahlen für Bahnen
 STR_2763    :???
 STR_2764    :Glückliche Gäste
 STR_2765    :Große Menschenmenge
-STR_2766    :???
+STR_2766    :Gewinne Szenario
```
Comment (Gymnasiast) [inline data/language/german.txt]: Doesn't the German translation use infinitives ("Szenario gewinnen") rather than imperatives?

**Comment by dkessel** [comment]
Right. it is "Szenario gewinnen" now.

**Comment by dkessel** [comment]
@Gymnasiast is this okay to be merged?

**Comment by Gymnasiast** [comment]
Yes, but I can't merge it for you. Only @runelaenen or @IntelOrca can do that.

**Comment by runelaenen** [comment]
I usually wait 3 days, if noone has complained by then I merge. (this is for languages I don't speak)

**Pull request merged**